back-dispatcher/routers/EverythingRoute/everythingModel.ts

```diff
@@ -0,0 +1,7 @@
+import mongoose from "mongoose";
+import { everythingSchema } from "./everythingSchema";
+
+export const everythingModel = mongoose.model(
+  "everythingArticle",
+  everythingSchema
+);
```

Comment:
put this code in the schema ts file.
the model ts file is handling  the functionality for example:
find, delete, ext.